railties/lib/rails/tasks.rb

```diff
@@ -5,7 +5,6 @@
 # Load Rails Rakefile extensions
 %w(
   framework
-  log
```

Comment:
Why we are not removing the `.rake` file that define this command?

Reply:
Thanks, I missed that.
I've removed the file.